lib/components/collapse.vue

```diff
@@ -43,10 +43,9 @@
         },
 
         watch: {
-            visible(newVal, oldVal){
-                if (newVal !== this.show) {
-                    this.toggle()
-                }
+            visible(newVal) {
+                this.show = newVal;
```

Comment:
Not to be picky, but then we have exactly what I had before where I just called `vm.toggle()`